XpringKit/XRP/Model/TransactionResult.swift

```diff
@@ -0,0 +1,19 @@
+/// Represents the outcome of submitting an XRPL transaction.
+public struct TransactionResult {
+
+  /// The identifying hash of the transaction.
+  public let hash: TransactionHash
+
+  /// The TransactionStatus indicating the outcome of this transaction.
+  public let status: TransactionStatus
+
+  /// Whether this transaction is included in a validated ledger.  The transactions status is only final if this field is true.
+  public let validated: Bool
+
+  /// Create a new TransactionResult object.
```

Comment:
You can actually omit this:
https://docs.swift.org/swift-book/LanguageGuide/Initialization.html#ID213

Swift will generate you initializers automatically if you use a struct and all the fields of the object are also structs or primitives.

Reply:
removed